rules/network/command_and_control_cobalt_strike_default_teamserver_cert.toml

```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2020/10/05"
+ecs_version = ["1.6.0"]
```

Comment:
```suggestion
```

Reply:
This is no longer required unless the rule won't work with the latest ECS version.